### PR TITLE
Readme documentation

### PR DESCRIPTION
**Pull request by ryanleesipes**
Updated README with a link to Adapt site for more in-depth examples and documentation.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 argparse==1.2.1
 pyee==0.1.0
 wsgiref==0.1.2
+pdoc==0.3.1
```
Comment (clusterfudge) [inline requirements.txt]: this does not belong in requirements.txt, as it's only a requirement for generating the docs, not for using the library.

Comment (ryanleesipes) [inline requirements.txt]: I committed that? Will pip add that automatically?

Comment (ryanleesipes) [inline requirements.txt]: Cuz I don't remember adding that. Lol.

Comment (clusterfudge) [inline requirements.txt]: pip doesn't, but pycharm may have.

**Comment by ryanleesipes** [comment]
Fixed.

**Comment by ryanleesipes** [comment]
@clusterfudge Look good to you?

**Comment by clusterfudge** [comment]
LGTM. Would you mind rebasing off master to confirm tests pass? It's just a text file update, so should be fine.

**Comment by ryanleesipes** [comment]
Updated, I may try to get the badges and all that good stuff wtihin the documentation soon. This needs more love and attention.

**Pull request merged**